inference-engine/thirdparty/clDNN/src/graph_optimizer/remove_redundant_reorders.cpp

```diff
@@ -343,8 +348,8 @@ void remove_redundant_reorders::run(program& p) {
         auto& dep = node->get_dependency(0);
         if (!(usr->is_type<convolution>()) ||
              (usr->get_output_layout().data_type != dep.get_output_layout().data_type) ||
-             (dep.get_output_layout().format != format::bfyx) ||
-             (usr->get_output_layout().format != format::fs_b_yx_fsv32))
+             (usr->get_output_layout().format != format::fs_b_yx_fsv32 && usr->get_output_layout().format != format::b_fs_yx_fsv32) ||
```

Comment:
This is now mismatching with the comment. Could you check the comment just above this line?

Reply:
I found that this code was initially inserted by @sshlyapn.
Sergey, do we have update in convolution code as well to support b_fs_yx_fsv32?